README.md

```diff
@@ -151,7 +151,7 @@ Fishes only can spawn between y:45 and y:63, both excluded.
 * Categories: `epsilon-carpet`, `survival`, `bugfix`
 
 ### antiEnderGriefExceptMelon
-Prevent endermans griefing blocks except melons
+Prevents endermans griefing blocks except melons
```

Comment:
Ok, if you *really* wanted to get technical with how this should be phrased it should probably be
"Prevents enderm**e**n **from** griefing **all** blocks except melons"